src/app.js

```diff
@@ -1 +1,44 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (source, destination) => {
+  if (!source || !destination) {
+    throw new Error('Source and destination must be provided');
```

Comment:
Consider providing more detailed error messages to help users understand what went wrong. For example, specify which parameter is missing.